import-dashboards.py

```diff
@@ -255,8 +255,19 @@ def rename_panels():
             if os.path.exists(panel_path):
                 with open(panel_path, 'r') as f:
                     panel_params = json.loads(f.read())
+                    if panel == panel_params['id']:
```

Comment:
Could you please add a comment on why we need to skip in this case?

Reply:
I have added more details into the output message. So it has the reason of using the checking.